modules/infoicon/Infoicon.story.tsx

```diff
@@ -1,9 +1,33 @@
 import React from "react";
 import Infoicon from "./Infoicon";
+import { ComponentRegistry } from "@diana-ui/base";
 import { ThemeStyleSheetFactory } from "@diana-ui/types";
+import { Avatar } from "@diana-ui/avatar";
+
+const StyledAvatar = Avatar.extendStyles(theme => ({
+  wrapper: {
+    borderColor: theme.colors.grey.greenish,
+    borderStyle: "solid",
+    boxSizing: "border-box",
+    flexShrink: 0,
+    height: 48,
+    width: 48
+  }
+}));
+
+ComponentRegistry.register(StyledAvatar, "Avatar");
```

Comment:
you can pass {register: true} to the extend styles and it would register over Avatar